scripts/build-win64.bat

```diff
@@ -1,6 +1,6 @@
 :: Check to ensure that the VSDev command prompt is in the below location
 echo "Starting VSDev Command Prompt"
-call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\Tools\VsDevCmd.bat"
+call "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\Tools\VsDevCmd.bat"
```

Comment:
Wouldn't it be possible to check for both, so we support both versions of VS Studio?

Reply:
Possible, but, not necessary since we can use 2017 to build older versions of SDA as well.